prow/plugins/trigger/push.go

```diff
@@ -42,6 +44,16 @@ func listPushEventChanges(pe github.PushEvent) []string {
 	return changedFiles
 }
 
+func createRefs(pe github.PushEvent) kube.Refs {
```

Comment:
Could this be in `pjutil` in a DRY way?

Reply:
`PullRequest` and `PushEvent` are different enough to require two distinct functions.

Reply:
Ok.